src/lib.rs

```diff
@@ -370,13 +370,38 @@ impl SteamId {
             AccountId(bits.m_comp.m_unAccountID())
         }
     }
+
+    /// Returns the formatted SteamID32 string for this steam id.
+    pub fn steamid32(&self) -> String {
+        let account_id = self.account_id().raw();
+        let last_bit: u8 = if account_id % 2 == 0 { 0 } else { 1 };
```

Comment:
`let last_bit = account_id & 1` is clearer, as we are specifically dealing with the bits and not the arithmetic properties of the ID.
```suggestion
        let last_bit = account_id & 1;
```